tests/app/controllers/test_tenant.py

```diff
@@ -9,6 +9,7 @@ class TestTenant(BaseTestController):
     path = "/api/v1/tenants"
     attrs = {
         "name": "Some Honky Tenant!",
+        "api_key": "2d4b9f2b-ad45-484f-a0a6-75393bacba4c",
```

Comment:
With the above change, this line (here and below) would have to become something like that:

```
import uuid
...
        "api_key": uuid.uuid4(),
```